### PR TITLE
Improve Package

### PR DESCRIPTION
**Pull request by disufi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@livingdocs/node-sdk",
```
Comment (peyerluk) [inline package.json]: The `name` is outdated and points to the old repo. Can you fix this in this PR as well?

Comment (peyerluk) [reply package.json]: Thanks @mokimo for pointing this out 👍

Comment (disufi) [reply package.json]: I am a bit confused, how should it be named instead? see https://www.npmjs.com/package/@livingdocs/node-sdk

Comment (disufi) [reply package.json]: Or shall we rename it entirely and republish it with another name?

Comment (mokimo) [reply package.json]: Commented on the outdated "homepage", which you already fixed further down - I think Lukas referred to that!

Comment (peyerluk) [reply package.json]: No, no. I commented on this. But I was wrong, you two were right ;)

**Pull request merged**